### PR TITLE
new: more strict naming for service definitions

### PR DESCRIPTION
**Pull request by taliamax**
Added more strict rules to service definition naming, and used these rules to infer if a token is an IP or a service during GetNet

### CODE REVIEW

**Comment by rdsharma** [comment]
Did we decide to abandon this one?

**Comment by taliamax** [comment]
We had a small conversation on whether it should still be around. I think it would be advisable to keep it but remove the strict requirement, since if we allow it as a first-pass we will still add some efficiency

File: capirca/lib/naming.py [hunk 1]
```diff
@@ -134,6 +134,7 @@ def __init__(self, naming_dir=None, naming_file=None, naming_type=None):
     self.unseen_networks = {}
     self.port_re = re.compile(r'(^\d+-\d+|^\d+)\/\w+$|^[\w\d-]+$',
                               re.IGNORECASE|re.DOTALL)
+    self.token_re = re.compile(r'(^[-_A-Z0-9]+$)')
```
Comment (rdsharma) [inline capirca/lib/naming.py:137]: Do we disallow uppercase on purpose? (I don't see a re.IGNORECASE here, or we could just do A-z instead of A-Z)

Comment (rdsharma) [reply capirca/lib/naming.py:137]: Friendly ping :)

Comment (taliamax) [reply capirca/lib/naming.py:137]: Whoops! I thought I replied to this. I'm updating this PR today

**Pull request merged**